sklearn/decomposition/tests/test_sparse_pca.py

```diff
@@ -313,3 +313,21 @@ def test_spca_early_stopping(global_random_seed):
         max_iter=100, tol=1e-6, max_no_improvement=100, random_state=global_random_seed
     ).fit(X)
     assert model_early_stopped.n_iter_ < model_not_early_stopped.n_iter_
+
+
+def test_sparse_pca_inverse_transform():
+    """Check that `inverse_transform` in `SparsePCA` and `PCA` are similar."""
+    rng = np.random.RandomState(0)
+    n_samples, n_features = 10, 5
+    X = rng.randn(n_samples, n_features)
+
+    n_components = 2
+    spca = SparsePCA(
```

Comment:
Should we have a test for `MiniBatchSparsePCA` as well?

Reply:
This test would not work with `MiniBatchSparsePCA` but I added a round-trip check when we don't reduce the dimentionality.